### PR TITLE
Better sql for creating event packet

### PR DESCRIPTION
**Pull request by reginafcompton**
This PR corrects the SQL query in `findEventAgendaPacket`, which did not include the `event_id` in the where clause. 

It handles part of this issue: https://github.com/datamade/metro-pdf-merger/issues/16

### CODE REVIEW

File: lametro/management/commands/compile_pdfs.py [hunk 1]
```diff
@@ -177,58 +173,53 @@ def findEventAgendaPacket(self, all_documents=False):
                 ON i.bill_id=d_bill.bill_id
                 INNER JOIN councilmatic_core_eventdocument as d_event
                 ON i.event_id=d_event.event_id
-                INNER JOIN councilmatic_core_bill AS b
-                ON d_bill.bill_id=b.ocd_id
+        '''
+
+        if all_documents:
+            query += '''
```
Comment (fgregg) [inline lametro/management/commands/compile_pdfs.py]: I find this type of thing really hard to follow. please write out both queries even if they are very similar.

**Comment by reginafcompton** [comment]
@fgregg - I improved the readability of the query. I am going to take one more look at it tomorrow, so further thoughts are welcome.

File: lametro/management/commands/compile_pdfs.py [hunk 2]
```diff
@@ -221,14 +176,56 @@ def findEventAgendaPacket(self, all_documents=False):
...
+
+            event_agendas = self.connection.execute(sa.text(query))
+        else:
+            # The compile script typically runs without an `all_documents` argument.
+            # In those cases, the query should only grab the related event_ids for new bill and event documents.
+            grab_events = '''
+            SELECT DISTINCT event_id
+            FROM councilmatic_core_eventagendaitem as i
+            INNER JOIN new_billdocument as bill_doc
+            ON i.bill_id=bill_doc.bill_id
```
Comment (fgregg) [inline lametro/management/commands/compile_pdfs.py]: I think this is a little easier to read, since I don't have to remember aliases.

```sql
SELECT DISTINCT event_id
FROM councilmatic_core_eventagendaitem
INNER JOIN new_billdocument
USING (bill_id)
...
```
if you don't like that, please use more verbose aliases.

File: lametro/management/commands/compile_pdfs.py [hunk 3]
```diff
@@ -221,14 +176,56 @@ def findEventAgendaPacket(self, all_documents=False):
...
+                            d_bill.url,
+                            d_bill.bill_id,
+                            d_event.url as event_agenda,
+                            CASE
+                            WHEN trim(lower(d_bill.note)) LIKE 'board report%' THEN '1'
+                            WHEN trim(lower(d_bill.note)) LIKE '0%' THEN 'z'
+                            ELSE trim(lower(d_bill.note))
+                            END AS note
+                        FROM councilmatic_core_billdocument AS d_bill
+                        INNER JOIN councilmatic_core_eventagendaitem as i
```
Comment (fgregg) [inline lametro/management/commands/compile_pdfs.py]: see my other note about aliases.

File: lametro/management/commands/compile_pdfs.py [hunk 4]
```diff
@@ -221,14 +176,56 @@ def findEventAgendaPacket(self, all_documents=False):
                     ON i.bill_id=d_bill.bill_id
                     INNER JOIN councilmatic_core_eventdocument as d_event
                     ON i.event_id=d_event.event_id
-                    INNER JOIN councilmatic_core_bill AS b
-                    ON d_bill.bill_id=b.ocd_id
-                    WHERE d_bill.bill_id in ( {} )
-                    ) AS subq
+                ) AS subq
```
Comment (fgregg) [inline lametro/management/commands/compile_pdfs.py]: This is a good candidate for a https://www.postgresql.org/docs/9.1/static/queries-with.html 

Let me know if you want some help writing that.

File: lametro/management/commands/compile_pdfs.py (hunk 2 above)
Comment (reginafcompton) [reply lametro/management/commands/compile_pdfs.py]: I like that a lot.

**Comment by reginafcompton** [comment]
@fgregg - I made use of USING and WITH statements, and I renamed aliases for clarity.
